snapshotTestContent/Authoring-MVP3-(created-Aug-8)/current_snapshots/snapshot_1000_0.json

```diff
@@ -46,7 +46,7 @@
               "mode": "100644",
               "type": "Blob",
               "value": {
-                "contents": "{\"usedRoutes\":[\"\"],\"gcData\":{\"gcNodes\":{\"/root\":[\"/_scheduler/cf344c73-866d-4898-8b7f-eef33576cd21\",\"/_scheduler\"],\"/cf344c73-866d-4898-8b7f-eef33576cd21\":[\"/_scheduler\"],\"/\":[\"/_scheduler/root\",\"/_scheduler/cf344c73-866d-4898-8b7f-eef33576cd21\"]}}}",
```

Comment:
it's a bit of a bummer to lose this coverage. is it possible to not lose it?

Reply:
Well we could by running GC in the `snapshot` API. But that API is deprecated and adding stuff to it just seems wrong.

Reply:
At minimum, we should open an issue to increase coverage of replay tool to much better reflect what we ship.
Also, should we instead be generating new version of the snapshot (instead of modifying existing ones) to ensure we can successfully read old files?

Reply:
We should definitely update these tests to start using `summarize` instead of `snapshot`. I will create a work item for this.

I am not sure what you mean by your last statement - "should we instead be generating new version of the snapshot (instead of modifying existing ones) to ensure we can successfully read old files". It moved the current snapshots (with gc data in summary) to src_snapshots/0.40.0 and updated the current_snapshots.

Reply:
Created an item for this - https://github.com/microsoft/FluidFramework/issues/6237

Reply:
Sorry, I missed that it's done in this PR - I meant creating prior versions in src_snapshots